### ISSUE TITLE
[FT] Add flow end reason in `pfring_ft_flow_value`

### ISSUE THREAD

**Issue by tadams42** [ntop/PF_RING#497]
Hi, would it be possible to get something like this in `pfring_ft_flow_value`?

[RFC 5102 - flowEndReason](https://tools.ietf.org/html/rfc5102#section-5.11.3)

```
   Description:
      The reason for Flow termination.  The range of values includes the
      following:

      0x01: idle timeout
            The Flow was terminated because it was considered to be
            idle.

      0x02: active timeout
            The Flow was terminated for reporting purposes while it was
            still active, for example, after the maximum lifetime of
            unreported Flows was reached.

      0x03: end of Flow detected
            The Flow was terminated because the Metering Process
            detected signals indicating the end of the Flow, for
            example, the TCP FIN flag.

      0x04: forced end
            The Flow was terminated because of some external event, for
            example, a shutdown of the Metering Process initiated by a
            network management application.
```

**Comment by cardigliano** [comment]
@tadams42 this seems to be definitely a useful information.

**Comment by cardigliano** [comment]
Implemented with b5c132fe9d9e3d374069b5f6e666eec7251133da
